server/views/pages/unexpectedArrivals/noExistingRecordsFound.njk

```diff
@@ -30,7 +30,7 @@
             }) }}
         </form>
         <p>
-            <a class="govuk-body govuk-link govuk-link--no-visited-state" href="" data-qa="choose-prisoner">
+            <a class="govuk-body govuk-link govuk-link--no-visited-state" href="/confirm-arrival/choose-prisoner" data-qa="choose-prisoner">
```

Comment:
Added missing href - perhaps this should have been done on a separate bug fix ticket but such a small fix hope this is ok.